multigeiger/transmission.cpp

```diff
@@ -46,11 +46,12 @@ void setup_transmission(const char *version, char *ssid) {
   #endif
 }
 
-void prepare_http(HTTPClient *http, const char *host) {
+void prepare_http(HTTPClient *http, const char *host, int xpin) {
   http->begin(host);
   http->addHeader("Content-Type", "application/json; charset=UTF-8");
   http->addHeader("Connection", "close");
   http->addHeader("X-Sensor", chipID);
+  http->addHeader("X-PIN", String(xpin));
 }
```

Comment:
please don't do X-Pin here, this is intended to be generic code to prepare a json http request, no matter which server we are talking to.

Reply:
OK, understood. Will change that.

Reply:
if the same applies to X-Sensor, please move that also.

Reply:
Hmmm, this is the only point in the whole transmission, where the sensor ID is sent, so I think we need this for every server.
Btw. how can i get rid of the unwanted files here? I have adjusted my .gitignore so that they are not committed next time.

Reply:
guess you could use `git rebase -i master` and then "e"dit the commit.

then, I guess it is `git rm --cached ...`.